dice.toml

```diff
@@ -0,0 +1,29 @@
+[Server]
+Addr = '0.0.0.0'
+Port = 7379
+KeepAlive = 300
+Timeout = 300
+MaxConn = 0
+ShardCronFrequency = 1000000000
+MultiplexerPollTimeout = 100000000
+MaxClients = 200
+MaxMemory = 0
+EvictionPolicy = 'allkeys-lfu'
+EvictionRatio = 0.4
+KeysLimit = 20000
```

Comment:
This can be 100k maybe, we can modify subscription chan to depend on a new configuration key so it doesn't reserve a very large buffer (right now it recycles this value).

Reply:
Okay will create a new key named `WatchChanBufLimit`. Ideally it should be in proportion with `KeysLimit` value which is used for eviction.

Reply:
It's not necessary for it to be related to keys limit. A single key can have a very large number of updates coming in - which wouldn't push the keyslimit at all, but would fill up the watchChan

Reply:
Created config `WatchChanBufSize` default value set to 20000